Dockerfile

```diff
@@ -10,20 +10,18 @@ COPY ./ /project
 WORKDIR /project
 ENV PATH /project/node_modules/.bin:$PATH
 ENV NODE_OPTIONS --max_old_space_size=4096
-
+RUN corepack enable
 RUN chown -R node .
 USER node
-
-RUN cp /project/app/.env.sample /project/app/.env \
-  && yarn
-
+RUN cp /project/app/.env.sample /project/app/.env
+RUN yarn
 USER root
 RUN chown -R node .
 USER node
 RUN yarn lerna run build
 
 
-FROM node:20-alpine as nodejsbuild
+FROM nikolaik/python-nodejs:python3.13-nodejs22-slim as nodejsbuild
```

Comment:
why not alpine or the official [node-slim image](https://hub.docker.com/layers/library/node/23-slim/images/sha256-75a933eb4b879c8c0ebe7f48f57a134e4d1b3b60cc662164037abaaae8766cb9)?

Reply:
This base image includes python which is needed for the importer tooling stuff. If I had used node-slim, I would have had to setup python by some other means, I thought I could just re-use an existing solution.